app.py

```diff
@@ -11,29 +11,26 @@
 
 @app.route("/", methods=["GET"])
 def root():
-    if request.method == "GET":
-        return escape("This is the root :)")
+    return escape("This is the root :)")
 
 @app.route("/person", defaults={'personnummer': None}, methods=['GET'])
 @app.route("/person/<personnummer>", methods=['GET'])
 def person(personnummer):
-    if request.method == 'GET':
-        if personnummer is not None:
-            personnummer = personnummer+".json"
-        person = Person(file_name=personnummer)
-        return jsonify(person.batch)
+    if personnummer is not None:
+        personnummer = personnummer+".json"
+    person = Person(file_name=personnummer)
+    return jsonify(person.batch)
 
 
 @app.route("/batch", methods=['GET'])
-def single():
-    if request.method == 'GET':
-        number_of_available_people = Person().get_number_of_files_in_directory()
-
-        all_people = [dict() for x in range(number_of_available_people)]
-        for i in range (number_of_available_people):
-            all_people[i] = Person(file_position_in_directory=i).batch
-        return jsonify(all_people)
-    
+def batch():
+    number_of_available_people = Person().get_number_of_files_in_directory()
+
+    all_people = [dict() for x in range(number_of_available_people)]
+    for i in range (number_of_available_people):
+        all_people[i] = Person(file_position_in_directory=i).batch
+    return jsonify(all_people)
+
```

Comment:
fjernet unødvendige if-setninger
methods=['GET'] sjekker allerede for reequest-type